src/app/view/components/login/login-component/login-container.component.ts

```diff
@@ -50,7 +50,7 @@ export class LoginContainer {
             .loginUser(
                 loginData.login,
                 loginData.password,
-                loginData.remenberMe,
+                loginData.rememberMe,
```

Comment:
Nice catch!

Reply:
Question is it mapping something on the backend, since the typo might be there too.

Reply:
Naa it is private  final boolean rememberMe;